components/Ui/UiCard.vue

```diff
@@ -32,21 +32,21 @@
           <h5 v-if="subtitle">
             {{ subtitle }}
           </h5>
-        </div>
-        <div class="cds--row">
-          <div v-if="hasTags(tags)" class="card__tags">
-            <bx-tag
-              v-for="tag in tags"
-              :key="tag"
-              class="card__tag"
-              type="purple"
-            >
-              {{ tag }}
-            </bx-tag>
+          <div class="card__tags">
+            <template v-if="hasTags(tags)">
```

Comment:
If you add this check here, when you have a primary tag and not secondary tags, you will see no tags. For advocates cards is working because instead of passing the primary tag as `primaryTag`, we pass it as `tags` (what is wrong).

Reply:
Updated in https://github.com/Qiskit/qiskit.org/pull/3442/commits/a1c9db3717e90bdfee51def9905d8adc7bca823e